test/extensions/filters/http/wasm/config_test.cc

```diff
@@ -33,7 +33,16 @@ class WasmFilterConfigTest : public TestBaseWithParam<std::string> {
   Api::ApiPtr api_;
 };
 
-INSTANTIATE_TEST_SUITE_P(Runtimes, WasmFilterConfigTest, testing::Values("wavm", "v8"));
+INSTANTIATE_TEST_SUITE_P(Runtimes, WasmFilterConfigTest,
+                         testing::Values(
+#if defined(ENVOY_WASM_V8) && defined(ENVOY_WASM_WAVM)
```

Comment:
This pattern repeats several times. Let's make it another macro on top of this.
Could be in another PR